extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -538,7 +546,13 @@ export class AmpIframe extends AMP.BaseElement {
     if (box.width > 10 && box.height > 10) {
       return false;
     }
-    return true;
+    // Iframe is not tracking iframe if open with user interaction
+    if (this.isInContainer_ === undefined) {
+      this.isInContainer_ = !!closest(this.element, element => {
```

Comment:
Would be faster to just do a DOM query, as in `closestBySelector(this.element, 'amp-lightbox|amp-sidebar')`. And even better is to define a shared class and do:
```
closestBySelector(this.element, 'i-amphtml-overlay');
```